Some subcommands of yahcli don't run standalone because of missing dependency

### Description

Running `yahcli` standalone (i.e., at the command line) fails for some commands because of a missing `log4j.core` dependency:

```
Caused by: java.lang.UnsupportedOperationException: No class provided, and an appropriate one cannot be found.
	at org.apache.logging.log4j.LogManager.callerClass(LogManager.java:573)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:598)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:585)
	at com.swirlds.common.crypto.Signature.<clinit>(Signature.java:44)
	... 17 more
```


### Steps to reproduce

1. Run the program `yahcli` with a command that emits logs.
  - Not sure which ones those are.  I found this when adding a _new_ command that emitted logs.

### Additional context

_No response_

### Hedera network

other

### Version

2022-11-13

### Operating system

macOS

--- a/test-clients/build.gradle.kts
+++ b/test-clients/build.gradle.kts
@@ -68,6 +68,7 @@ dependencies {
         exclude("javax.annotation", "javax.annotation-api")
     }
     implementation(libs.headlong)
+    implementation(libs.log4j.core)
     implementation(testLibs.json)
     implementation(testLibs.junit.jupiter.api)
     implementation(testLibs.picocli)
@@ -100,7 +101,8 @@ tasks.shadowJar {
 
     manifest {
         attributes(
-            "Main-Class" to "com.hedera.services.bdd.suites.SuiteRunner"
+            "Main-Class" to "com.hedera.services.bdd.suites.SuiteRunner",
+            "Multi-Release" to "true"
         )
     }
 }
@@ -120,7 +122,8 @@ val yahCliJar = tasks.register<ShadowJar>("yahCliJar") {
 
     manifest {
         attributes(
-            "Main-Class" to "com.hedera.services.yahcli.Yahcli"
+            "Main-Class" to "com.hedera.services.yahcli.Yahcli",
+            "Multi-Release" to "true"
         )
     }
 }

